pl_examples/hydra_examples/conf/model/lr_scheduler/cosine.yaml

```diff
@@ -0,0 +1,6 @@
+# @package _group_
```

Comment:
all of the type definitions and basic defaults are better done as Structured Configs.

Reply:
Slightly confused on how to do this one, I see the example here: https://hydra.cc/docs/next/patterns/objects using cls as a str param and then passing in params as a dict. We also want type checking on the param arguments, what is the best way to structure this?

Reply:
Yeah, I need to document instantiate with Structured Configs.
created [this](https://github.com/facebookresearch/hydra/issues/701) as a reminder.

In the mean time, take a look at this:
https://github.com/facebookresearch/hydra/blob/master/plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py#L115-L128

Reply:
Example shown here: https://github.com/anthonytec2/pytorch-lightning/pull/1/commits/3d9a11c0ea7670080e4259c4e4e850e9979a9bfa

Reply:
What is the best way to handle something like: 
```
base_lr (float or list) – Initial learning rate which is the lower boundary in the cycle for each parameter group.
```

Reply:
> What is the best way to handle something like:
> 
> ```
> base_lr (float or list) – Initial learning rate which is the lower boundary in the cycle for each parameter group.
> ```

For now specify it as Any and do the type checking yourself at runtime somehow.